cmd/utils/nodecmd/nodeflags.go

```diff
@@ -241,6 +241,7 @@ var KSCNFlags = []cli.Flag{
 	utils.VTRecoveryIntervalFlag,
 	utils.ServiceChainNewAccountFlag,
 	utils.ServiceChainAnchoringFlag,
+	utils.DefaultTxGasLimitFlag,
```

Comment:
Is `kscnd` the only binary the setting is needed? I think other node types that have `SubBridgeFlag` also needs the new flag, also.

Reply:
This flag configures gaslimit of parent operator and child operator of ServiceChain. So I'm this flag is really needed at other node types such as `SPN` and `SEN`

Reply:
I added. Thanks for suggestion.